Sources/MuxPlayerSwift/Monitoring/Monitor.swift

```diff
@@ -20,67 +20,60 @@ class Monitor {
 
     func setupMonitoring(
         playerViewController: AVPlayerViewController,
-        options: MonitoringOptions
+        playbackID: String,
+        options: MonitoringOptions,
+        usingDRM: Bool = false
     ) {
 
-        let monitoredPlayer: MonitoredPlayer
+        let customerData: MUXSDKCustomerData
 
-        if let customerData = options.customerData {
+        if let externallySpecifiedCustomerData = options.customerData {
 
-            let customerDataCopy = MUXSDKCustomerData()
+            let modifiedCustomerData = MUXSDKCustomerData()
 
-            if let customerVideoData = customerData.customerVideoData {
+            if let customerVideoData = externallySpecifiedCustomerData.customerVideoData {
                 let customerVideoDataCopy = MUXSDKCustomerVideoData()
```

Comment:
question(blocker): Is this right? it looks like here and below you've left `customerVideoDataCopy` (and equivalent) code but those values aren't used by anything?

Reply:
There is a bug in this (`AVPlaverViewController`) overload, good spot.

This handles the case when the SDK gets a custom `MUXSDKCustomerData` to pass along to Mux Data. For passing through custom dimensions like video title for example. `MUXSDKCustomerData` is a reference type. The SDK makes a defensive deep copy to avoid changes to player software name / version by client code.

Here `customerVideoDataCopy` was a deep copy of `customerVideoData` and was supposed to be set on `modifiedCustomerData` (also a deep copy). The original value got passed through instead. The variable names were too similar so I also made them easier to tell apart.

A test to illustrate: https://github.com/muxinc/mux-player-swift/blob/77f2d44adf160e4eb5ff4bc8220be1cb98d2e840/Tests/MuxPlayerSwift/MonitorTests.swift#L1117

Reply:
There is a bug in this (`AVPlaverViewController`) overload, good spot.

This handles the case when the SDK gets a custom `MUXSDKCustomerData` to pass along to the AVPlayer monitoring SDK. `MUXSDKCustomerData` is a reference type. The SDK makes a defensive deep copy to prevent changes to the player software name / version by client code out of band.

Here: `customerVideoDataCopy` was a deep copy of `customerVideoData` and was supposed to be set on `modifiedCustomerData` (itself also a deep copy). The original video data value passed through instead. The variable names were too similar so I cleaned them up some so the copy is more apparent.

A test to illustrate (the checks are inside the `validate` call): https://github.com/muxinc/mux-player-swift/blob/77f2d44adf160e4eb5ff4bc8220be1cb98d2e840/Tests/MuxPlayerSwift/MonitorTests.swift#L1117

Reply:
Good reminder not to shortchange readability haha